articles/github-copilot-azure/TOC.yml

```
@@ -0,0 +1,27 @@
+items:
+- name: GitHub Copilot for Azure Preview
+  href: index.yml
+  items:
+    - name: Overview
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)